### PR TITLE
Build: Enables end-to-end tests in build-master workflow

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ jobs:
   end-to-end-test:
       docker:
         - image: circleci/node:8-browsers
-        - image: grafana/grafana:master
+        - image: grafana/grafana-dev:$CIRCLE_SHA1
```
Comment (xlson) [inline .circleci/config.yml]:
```suggestion
        - image: grafana/grafana-dev:master-$CIRCLE_SHA1
```

See: https://hub.docker.com/r/grafana/grafana-dev/tags

**Review by xlson — CHANGES_REQUESTED** [review]
Looks good, just need to fix the docker image name.

**Review by ryantxu — COMMENTED** [review]
Looks fine to me -- but I'm guess I was requested for review by mistake? :)

File: .circleci/config.yml (hunk 1 above)
Comment (hugohaggmark) [reply .circleci/config.yml]: Good catch! I'll fix

**Review by xlson — APPROVED** [review]
LGTM!

**Pull request merged**